### DISCUSSION TITLE
May I know how to center hover window?

### DISCUSSION THREAD

**Discussion by nmsobri** [nvimdev/lspsaga.nvim#987] [Q&A]
As title, is there a way to center the hover window vertically and horizontally center?

**Comment by glepnir** [comment]
cursor center?

**Reply by nmsobri** [reply]
no, hover window itself

**Comment by glepnir** [comment]
now it only support in right top and cursor.

**Reply by nmsobri** [reply]
ah that's sad.. can you add that option? anyway can you link me to the documentation regarding how to change it position as u said
